src/session_io.cpp

```diff
@@ -853,11 +853,11 @@ std::string SessionIO::parseCodeMapGenerics(const std::string &screen, const std
 
 
 /**
- * @brief Parses string and returns code mapping and positions
+ * @brief Parses string and returns code mapping and positions per expression
  * @param sequence
  * @return
  */
-std::vector<MapType> SessionIO::pipe2codeMap(const std::string &sequence, const std::string &expression)
+std::vector<MapType> SessionIO::parseToCodeMap(const std::string &sequence, const std::string &expression)
```

Comment:
Switch to a more generic name since it's used by multiple methods and functions now.